### ISSUE TITLE
README dependency links broken

### ISSUE THREAD

**Issue by brian-amos-embedded** [adafruit/Adafruit_Python_Extended_Bus#14]
[Adafruit Python](https://github.com/adafruit/Python)
[the Adafruit library and driver bundle](https://python.org/libraries)

are both 404 errors

Is https://pypi.org/project/adafruit-circuitpython-adafruitio/ potentially the updated library bundle?

**Comment by makermelissa** [comment]
Yeah, that's wrong... I think Blinka is really the only dependency.

**Comment by PaulskPt** [comment]
I installed this repo. Using it with a Raspberry Pi 5B-8GB and an Adafruit AHT20 sensor. Works great.
I found the link to this repo while studying the Adafruit-Blinka repo because i2c bus #1 suddenly did nothing anymore. dmesg gave a lot of 'lost arbritration' errors.

I managed to add defenitions for an I2C bus #3. For this I added SDA2 / SCL2 definitions in two adafruit-blinka repo files (in /board/raspberrypi/rpi_5b.py and in /microcontroller/bcm2712/pin.py and in /boot/firmware/config.txt).  This worked OK. 
When I encountered the link to this repo, I installed it and tested it. Implementation of this repo is a more simple  solution (for the user, me) than my 'workaround'.  

Thank you for creating this repo!
